tools/release-plugin.sh

```diff
@@ -321,16 +321,91 @@ gh pr create --title "Backport $PLUGINS_CHANGED Changes" --body "$(cat .github/f
 rm .github/files/TEMP_BACKPORT_RELEASE_CHANGES.md
 
 yellow "Release script complete!"
-cat <<"EOM"
 
-Next you need to merge the above PR into trunk.
+echo ''
```

Comment:
Any nuanced difference in echoing an empty string vs omitting it?

Reply:
Not that I know of. It just makes more sense to me this way for some reason.